ecommerce/serializers/subcategories.py

```diff
@@ -0,0 +1,79 @@
+from rest_framework import serializers
+
+from ecommerce.models import ProductSubCategory
+
+
+class SubCategoryListSerializer(serializers.ModelSerializer):
+    """
+    Serializer for listing subcategories.
+
+    This serializer provides essential information for subcategory listings,
+    optimized for performance and minimal data transfer.
+    """
+
+    category_name = serializers.CharField(source="category.name", read_only=True)
```

Comment:
We should use `category` instead of `category_name`. When working with FE or a mobile client, sometimes using just the `name` field is not enough.

Better to provide some basic information, at least the `name` and `id`.